templates/public.php

```diff
@@ -76,11 +76,11 @@
 						  data-owner="<?php p($_['displayName']) ?>"
 						  data-name="<?php p($_['filename']) ?>">
 									<button id="save-button"><?php p(
-											$l->t('Add to your ownCloud')
+											$l->t('Add to your %s', [(new \OCP\Defaults())->getName()])
 										) ?></button>
 									<form class="save-form hidden" action="#">
 										<input type="text" id="remote_address"
-											   placeholder="example.com/owncloud"/>
+											   placeholder="cloud.example.com"/>
```

Comment:
I agree with @LukasReschke . I think it's bad practice to use a subfolder, especially if other apps are installed in other folders.